packages/ckb-sdk-utils/__tests__/crypto/blake2b.test.js

```diff
@@ -47,3 +48,54 @@ describe('blake2b', () => {
     }
   })
 })
+
+describe('blake2b-wasm-ready', () => {
+  afterEach(() => {
+    delete globalThis.navigator
+  })
+
+  describe("When it's in browser but not iOS 11", () => {
+    beforeEach(() => {
+      Object.defineProperty(globalThis, 'navigator', {
+        value: {
+          userAgent: 'Mozilla/5.0 (iPhone; CPU iPhone OS 12_4_4 like Mac OS X)',
+        },
+      })
+    })
+
+    it('callback should be invoked without error', () => {
+      expect.assertions(1)
+      ready(err => {
+        expect(err).toBeUndefined()
+      })
+    })
+  })
+
+  describe("When it's not in browser and navigator is undefined", () => {
+    beforeEach(() => {
+      delete globalThis.navigator
+    })
+    it('callback should be invoked without error', () => {
+      expect.assertions(1)
+      ready(err => {
+        expect(err).toBeUndefined()
+      })
+    })
+  })
+
+  describe("When it's iOS 11", () => {
+    beforeEach(() => {
+      Object.defineProperty(globalThis, 'navigator', {
+        value: {
+          userAgent: 'Mozilla/5.0 (iPhone; CPU iPhone OS 11_2_1 like Mac OS X)',
+        },
+      })
+    })
+    it('callback should be invoked with error', () => {
+      expect.assertions(1)
+      ready(err => {
+        expect(err).toEqual(new Error('blake2b-wasm is unavailable on iOS 11'))
```

Comment:
If platform is iOS 11 , will it throw an error not use blaks2b.js ?

Reply:
> If platform is iOS 11 , will it throw an error not use blaks2b.js ?

An error will be thrown when `ready`(which used with `blake2b-wasm`) is used, but no errors will be thrown by just using the blake2b.js